### PR TITLE
Updated metadata + Screenshots

### PR DESCRIPTION
**Pull request by mayankkuthar**

### CODE REVIEW

File: activity/activity.info [hunk 1]
```diff
@@ -6,6 +6,6 @@ icon = activity-help
 exec = sugar-activity helpactivity.HelpActivity
 license = GPLv3
```
Comment (vipulgupta2048) [inline activity/activity.info:7]: `GPLv3` will be `GPLv3+`

**Review by vipulgupta2048 — CHANGES_REQUESTED** [review]
Thanks for the pull request @Gabbar8055 
From what I am seeing, the help activity is being maintained here - https://github.com/godiard/help-activity
Please make a PR there.

**Comment by quozl** [comment]
Closing because @Gabbar8055 forgot to do so after opening https://github.com/godiard/help-activity/pull/103

**Comment by vipulgupta2048** [comment]
Thanks @Quozl

**Pull request closed without merging**